src/main/java/org/rumbledb/compiler/VisitorConfig.java

```diff
@@ -91,12 +108,14 @@ public boolean suppressErrorsForAccessingUnsetExecutionModes() {
         .withsuppressErrorsForFunctionSignatureCollision(true)
         .withSuppressErrorsForCallingMissingFunctions(false)
         .withSuppressErrorsForAccessingUnsetExecutionModes(false)
+        .withSetUnsetToLocal(true)
```

Comment:
Previously, we would set all unset function identifiers to local(without using a visitor) to solve deadlocks and then make a final visitor pass.

I wanted to clarify the current state after seeing this new flag `setUnsetToLocal`:
It seems that this flag is only used to set unset variable references to local. If this is the case I would recommend updating the name to indicate this. 

Furthermore, (I feel my brain is a bit bloated with information so I may be missing some important details while asking the following two questions):
- Do we still set the functions from Unset to Local "prior" to the final pass? Or do we do it during the final pass?
- In what scenerio do variables require a forced execution more from unset to local?

Reply:
> I wanted to clarify the current state after seeing this new flag `setUnsetToLocal`:
> It seems that this flag is only used to set unset variable references to local. If this is the case I would recommend updating the name to indicate this.

Yes, that is the case and this is renamed.

Reply:
> * Do we still set the functions from Unset to Local "prior" to the final pass? Or do we do it during the final pass?

Yes, we still do. This code is still here.